src/gui/transferlistwidget.cpp

```diff
@@ -260,7 +260,7 @@ TransferListWidget::TransferListWidget(QWidget *parent, MainWindow *mainWindow)
 
     //Ensure that at least one column is visible at all times
     bool atLeastOne = false;
-    for (unsigned int i = 0; i<TorrentModel::NB_COLUMNS; i++) {
+    for (unsigned int i = 0; i<TorrentModel::NB_COLUMNS; ++i) {
```

Comment:
Can you also add spaces around `<` operator?
And below too.

Reply:
will do.